{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -83,6 +83,15 @@
         # The node container isn't started (yet?)
         pass
     {%- endif %}
+{% if cookiecutter.windows == 'y' %}
+# RunServerPlus
+# ------------------------------------------------------------------------------
+# After how many seconds auto-reload should scan for updates in poller-mode
+RUNSERVERPLUS_POLLER_RELOADER_INTERVAL = 5
+
+# Werkzeug reloader type [auto, watchdog, or stat]
+RUNSERVERPLUS_POLLER_RELOADER_TYPE = 'stat'
+{% endif %}
```

Comment:
To avoid leaving an extra empty line:

```suggestion
{%- endif %}
```